src/operations/SymbolicOperation.cpp

```diff
@@ -312,6 +312,14 @@ bool SymbolicOperation::equals(const Operation& op, const Permutation& perm1,
   throw QFRException("OpenQASM 2.0 doesn't support parametrized gates!");
 }
 
+[[noreturn]] void
+SymbolicOperation::dumpOpenQASM3([[maybe_unused]] std::ostream& of,
+                                 [[maybe_unused]] const qc::RegisterNames& qreg,
+                                 [[maybe_unused]] const qc::RegisterNames& creg,
+                                 [[maybe_unused]] uint32_t indent) const {
+  throw QFRException("OpenQASM 2.0 doesn't support parametrized gates!");
+}
+
```

Comment:
First of all, the error message is kind of wrong here.
Second, OQ3 does support symbolic gates. However, I guess supporting these might be a little tough at the moment (need to add parameters of the circuit to the overall OQ3 program, etc.)
Depending on how you feel about it, either just change the error message here to something like "... not yet supported ... " or try to implement the parameters.